doc/source/user_guide/missing_data.rst

```diff
@@ -942,3 +943,25 @@ work with ``NA``, and generally return ``NA``:
    in the future.
 
 See :ref:`dsintro.numpy_interop` for more on ufuncs.
+
+.. _missing_data.NA.conversion:
+
+Conversion
```

Comment:
you may want to add a link to some of the IO routines (e.g. read_csv) to this. (and possibly mention in the doc-strings as well)

Reply:
I've added links to `read_csv` and `read_excel` in this section.

@jreback When you say "mention in the doc-strings", you mean in the doc-strings of `read_csv` and `read_excel` (for example), or somewhere else?